apps/omg_watcher/test/integration/in_flight_exit_test.exs

```diff
@@ -295,8 +295,7 @@ defmodule OMG.Watcher.Integration.InFlightExitTest do
              "byzantine_events" => [
```

Comment:
how about the comment we talked about in the upper assertion? (why only one pb available?)